spec/controllers/api/v1/offers_controller_spec.rb

```diff
@@ -477,9 +477,30 @@
     let(:schedule1) { create :schedule, scheduled_at: Time.now + 1.days }
     let(:delivery) { create :delivery, offer: scheduled_offer, schedule: schedule }
     let(:delivery1) { create :delivery, offer: scheduled_offer1, schedule: schedule1 }
+    let(:shareable1) {create :shareable, resource: reviewing_offer}
+    let(:shareable2) {create :shareable, resource: submitted_offer}
     before(:each) { generate_and_set_token(reviewer) }
     subject { JSON.parse(response.body) }
 
+    context "Shareable Offers/Published offers filter" do
+      before(:each){
+        shareable1
+        shareable2
+      }
```

Comment:
Remove this and use `let!`, which creates the db record instantly

let!(:shareable1) {create :shareable, resource: reviewing_offer}
let!(:shareable2) {create :shareable, resource: submitted_offer}